Automattic/newspack-theme
# SCSS - set up linting & formatting

### All Submissions:






### Changes proposed in this Pull Request:



This sets up linting (stylelint) and formatting (prettier). Also configures the style linter to run in pre-commit hook - to make it work, update the hook by running `$ ./vendor/bin/cghooks update`

### How to test the changes in this Pull Request:

1. look at the code changes - these should all be refactoring
2. run `npm run lint:scss` - there should be no errors
3. run `npm run format:scss` - there should be changes made to files

### Other information:
